chainerx_cc/chainerx/routines/activation.cc

```diff
@@ -18,13 +15,29 @@
 #include "chainerx/routines/creation.h"
 #include "chainerx/routines/explog.h"
 #include "chainerx/routines/indexing.h"
+#include "chainerx/routines/manipulation.h"
 #include "chainerx/routines/misc.h"
 #include "chainerx/routines/type_util.h"
 #include "chainerx/scalar.h"
 #include "chainerx/shape.h"
 
 namespace chainerx {
 
+Array ClippedRelu(const Array& x, Scalar z) {
+    Dtype dtype = internal::GetMathResultDtype(x.dtype());
+    const Array& x_cast = x.dtype() == dtype ? x : x.AsType(dtype);
+    return Minimum(Maximum(0, x_cast), z);
+}
+
+Array Crelu(const Array& x, int8_t axis) {
```

Comment:
Maybe it should be CRelu, according to existing ChainerX "ReLU-conventions"? Not a strong opinion and I can fix it in a separate PR if you want.

Reply:
Sure @hvy . I'll fix it.